assets/js/analyticsGraphs.js

```diff
@@ -0,0 +1,10 @@
+$(() => {
```

Comment:
if modern js proves too limiting, i'd suggest we use typescript for FE too and compile to whatever the appropriate version is

Reply:
i've already removed IE8-specific support (which didn't work anyway) but "modern" js might be too modern